src/test/java/Homework23.java

```diff
@@ -0,0 +1,65 @@
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.BasePage;
+import pages.HomePage;
+import pages.LoginPage;
+import org.openqa.selenium.By;
+
+public class Homework23 extends BaseTest {
+
+    @Test(enabled = true, priority = 0, description = "Login with valid email and password")
+
+    public void loginWithValidCredentialsTest(){
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+        loginPage.login();
+        Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
+    }
+    @Test(enabled = true, priority = 1, description = "Validate can create a playlist")
+    public void createNewPlaylistTest(){
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+        loginPage.login();
+        homePage.createPlaylist();
+        Assert.assertTrue(homePage.newPlaylistIsDisplayed());
+    }
+    @Test(enabled = true, priority = 2, description = "Validate can add a song to playlist")
+    public void addSongToExistingPlaylistTest()  {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+        loginPage.login();
+        homePage.searchASong("Lost In Space")
+                .clickViewAllButton()
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/225230796-0690fbb5-2215-4789-80d1-fc9ca7bc8252.png)